contributing/building-an-amp-extension.md

```diff
@@ -715,6 +715,8 @@ enable your experiment.
 If you are testing on your localhost, use the command `AMP.toggleExperiment(id,
 true/false)` to enable the experiment.
 
+File a github issue to cleanup your experiment. Assign it to yourself as a  reminder to remove your experiment and code checks. The removal of the experiment happens when your extension has been thoroughly tested and you address any feedback from those who are testing it.
```

Comment:
Two small nits:
- There are two spaces between "a" and "reminder" 
- I would word the second sentence slightly different: "Removal of your experiment happens after the extension has been thoroughly tested and all issues have been addressed."

Reply:
fixed. Thanks Crystal!